README.md

```diff
@@ -0,0 +1,72 @@
+| `projectId`      | project id to auth in a client     | true     | conduit-pubsub                                                                 |
+| `subscriptionId` | subscription name to pull messages | true     | conduit-subscription                                                           |
+
+### Destination
+A destination connector represents an **asynchronous** writes to the GCP Pub/Sub.
+
+`Configure` parses the configuration and validates them.
+
+`Open` initializes the GCP Pub/Sub client.
+
+`WriteAsync` publishes the record to the GCP Pub/Sub topic asynchronously. Messages are batched and sent according to `batchSize` and `batchDelay` parameters in the configuration settings.
+
+`Flush` does nothing, because the system does not cache messages before sending them.
+
+`Teardown` cancels the context, sends all remaining published messages, and releases the GCP Pub/Sub client.
+
+#### Configuration
+The user can get the authorization data from a JSON file by the following instructions: [Getting started with authentication](https://cloud.google.com/docs/authentication/getting-started).
+
+| name          | description                                                                                    | required | example                                                                        |
+|---------------|------------------------------------------------------------------------------------------------|----------|--------------------------------------------------------------------------------|
+| `privateKey`  | private key to auth in a client                                                                | true     | -----BEGIN PRIVATE KEY-----\nMIIEvAIBADANBgkqhkiG\n-----END PRIVATE KEY-----\n |
+| `clientEmail` | client email to auth in a client                                                               | true     | test_user@conduit-pubsub.iam.gserviceaccount.com                               |
+| `projectId`   | project id to auth in a client                                                                 | true     | conduit-pubsub                                                                 |
+| `topicId`     | topic name to push messages                                                                    | true     | conduit-topic                                                                  |
+| `batchSize`   | the size of the batch of messages, on completing which the batch of messages will be published | false    | 10                                                                             |
+| `batchDelay`  | the time delay, after which the batch of messages will be published                            | false    | 100ms                                                                          |
+
+### Known limitations
+The maximum message size in a publish request is **10Mb**.
+
+[Pub/Sub quotas and limits](https://cloud.google.com/pubsub/quotas)
```

Comment:
Is this intended to go into into a "references" section, or maybe in relation to the max. message size?

Reply:
Yes, it's about a message limitation.

[7a84b0b](https://github.com/conduitio-labs/conduit-connector-gcp-pubsub/commit/7a84b0b3c0d4b3346e1da41b23a543fc82040828)

Reply:
:+1:

Reply:
:+1: